tgui/packages/tgui/interfaces/PreferencesMenu/JobsPage.tsx

```diff
@@ -134,10 +134,19 @@ const JobRow = (
 
   const experienceNeeded = data.job_required_experience && data.job_required_experience[name];
   const daysLeft = data.job_days_left ? data.job_days_left[name] : 0;
+  const isLocked = job.locked;
 
   let rightSide: InfernoNode;
 
-  if (experienceNeeded) {
+  if (isLocked) {
+    rightSide = (
+      <Stack align="center" height="100%" pr={1}>
+        <Stack.Item grow textAlign="right">
+          Locked by the server & map configuration
+        </Stack.Item>
+      </Stack>
+    );
+  } else if (experienceNeeded) {
```

Comment:
Jobs that are disabled in the server config will never appear and shouldn't show at all. Only jobs disabled by the map should appear and it should say "Not available on this map" or something

Reply:
![image](https://github.com/BeeStation/BeeStation-Hornet/assets/87972842/5e7ad1f4-16ff-488e-85ce-e17afe58dc49)